src/openmc_nek_driver.cpp

```diff
@@ -241,6 +244,22 @@ void OpenmcNekDriver::init_volumes()
       openmc_driver_->comm_.Bcast(elem_volumes_.data(), n_global_elem_, MPI_DOUBLE);
     }
   }
+
+  // Volume check
+  if (this->has_global_coupling_data()) {
+    for (gsl::index i = 0; i < openmc_driver_->cells_.size(); ++i) {
+      const auto& c = openmc_driver_->cells_[i];
+      double v_openmc = c.volume_;
+      double v_nek = 0.0;
+      for (const auto& elem : cell_to_elems_.at(i)) {
+        v_nek += elem_volumes_.at(elem);
+      }
+      std::stringstream msg;
+      msg << "Cell " << openmc::model::cells[c.index_]->id_ << " (" << c.instance_
+          << "), V = " << v_openmc << " (OpenMC), " << v_nek << " (Nek)";
+      comm_.message(msg.str());
+    }
+  }
```

Comment:
@paulromano I'm also happy to merge this as-is, in order to get the bug fix in ASAP.  I can add a separate issue for the warning message.  I know you're busy with the report.  Let me know.

Reply:
I thought about that, but the issue is I generally don't assign volumes to all materials in the OpenMC model because it's not super straightforward (hoping to improve that in the future). The only volumes that are actually used by ENRICO are the ones for materials that have heat production in them. Right now, all the others I'm setting to one just to avoid errors. So, they will definitely have large errors and a warning won't be that useful. I'm using the output more as a sanity check for volumes that I _know_ should match.

Reply:
Understood, thanks!